zebra-network/src/peer_set/set.rs

```diff
@@ -282,6 +287,9 @@ where
     /// - `inv_stream`: receives inventory changes from peers,
     ///                 allowing the peer set to direct inventory requests;
     /// - `address_book`: when peer set is busy, it logs address book diagnostics.
+    /// - `minimum_peer_version`: endpoint to see the minimum peer protocol version in real time.
```

Comment:
Technically minimum peer version updates are delayed until blocks are verified, even if the network has a higher minimum

```suggestion
    /// - `minimum_peer_version`: endpoint to see the minimum peer protocol version based on the latest state tip height.
```

Reply:
This isn't a blocker so I'm just going to merge the PR